### ISSUE TITLE
Suggestion: create new rule "new line between class members" 

### ISSUE THREAD

**Issue by vyshkant** [palantir/tslint#3777]
I propose to create a new rule that would allow you us track new lines between class members.

Let's call it `newline-between-member`. The minimum set of properties is: `["never", "always"]`.

When `never` is applied, there should not be new lines between class members:
```typescript
class A {
    private foo: string;
    private bar: number;
    constructor() {
    }
    public doSomething(): void {
        // do something
    }
    public doSomethingElse(): void {
        // do something else
    }
}
```

Conversely, when `always` is applied, there should be a new line between class members:
```typescript
class A {

    private foo: string;

    private bar: number;

    constructor() {
    }

    public doSomething(): void {
        // do something
    }

    public doSomethingElse(): void {
        // do something else
    }
}
```

**Comment by ball6847** [comment]
I would love to see this rule too.

Also, it should be configurable to apply the rule to which members type, eg. method, property.
